input/pagecontent/reference.md

```diff
@@ -1010,20 +1010,26 @@ The referenced invariant and its properties must be declared somewhere within th
 
 #### Mapping Rules
 
-🚧 [Mappings](https://www.hl7.org/fhir/mappings.html) are an optional part of SDs that can be provided to help implementers understand the content and use resources correctly. These mappings are informative and are not to be confused with the computable mappings provided by [FHIR Mapping Language](https://www.hl7.org/fhir/mapping-language.html) and the [StructureMap resource](https://www.hl7.org/fhir/structuremap.html).
+[Mappings](https://www.hl7.org/fhir/mappings.html) are an optional part of SDs that can be provided to help implementers understand the content and use resources correctly. These mappings are informative and are not to be confused with the computable mappings provided by [FHIR Mapping Language](https://www.hl7.org/fhir/mapping-language.html) and the [StructureMap resource](https://www.hl7.org/fhir/structuremap.html).
 
 In FSH, mapping rules are not included in the profile definition.  Rather, they are included in a separate [Mapping definition](#defining-mappings) that provides additional context such as the higher level source and target. Within that definition mapping rules use the symbol `->` with the following grammar:
 
 ```
-* {path} -> {string}
+* -> "{map}" "{comment}" {mime-type}
+```
+
 ```
+* {path} -> "{map}" "{comment}" {mime-type}
+```
+
+In these rules, the `"{comment}"` and `{mime-type}` are optional. The mime type must conform to https://tools.ietf.org/html/bcp13 (FHIR value set https://www.hl7.org/fhir/valueset-mimetypes.html).
 
 **Examples:**
 
 * Map the entire profile to a Patient item in another specification:
 
   ```
-  * -> Patient
+  * -> "Patient" "This profile maps to Patient in Argonaut"
```

Comment:
I don't think this is an accurate description of what this does. Leaving an empty path does not map the entire profile. When you defined a mapping:
```
Mapping: MyMap
Source: MyPatient
Target: "http://someotherpatient.com"
```
Defining that mapping on `MyPatient` is what maps the entire profile. This is what adds information to the `mapping` field on the `MyPatient` StructureDefinition itself. Adding a rule like:
```
* -> "Patient"
```
adds a mapping to the top level `ElementDefinition` in the `snapshot` array on the `MyPatient` SD

Reply:
Note that for all rules, applying a rule without specifying a path:
```
* {someRule}
```
will apply that rule to the top level `ElementDefinition` in the `snapshot`, since our syntax for paths assumes that the resource name is implied, aka a FHIR path is `Patient.gender`, but the FSH path is only `gender`. The only exception to this is currently `^` rules. We've noticed this inconsistency, and in talking to @cmoesel about it, we decided that it would be better to keep the current behavior for now, and resolve this inconsistency post ballot.